browser/sources.gni

```diff
@@ -367,5 +368,6 @@ brave_chrome_browser_deps += brave_browser_speedreader_deps
 brave_chrome_browser_deps += brave_browser_themes_deps
 brave_chrome_browser_deps += brave_chromium_src_chrome_browser_deps
 brave_chrome_browser_deps += brave_chromium_src_chrome_browser_prefs_deps
+brave_chrome_browser_deps += brave_chromium_src_chrome_browser_first_run_deps
```

Comment:
to be sorted it should go after ```brave_chromium_src_chrome_browser_deps``` but before ```brave_chromium_src_chrome_browser_prefs_deps```.

Reply:
Just deleted because no new deps are added in `first_run_internal_posix.cc`.